cmd/dockerd/options.go

```diff
@@ -72,7 +73,8 @@ func (o *daemonOptions) InstallFlags(flags *pflag.FlagSet) {
 	flags.Var(opts.NewQuotedString(&tlsOptions.KeyFile), "tlskey", "Path to TLS key file")
 
 	hostOpt := opts.NewNamedListOptsRef("hosts", &o.Hosts, opts.ValidateHost)
-	flags.VarP(hostOpt, "host", "H", "Daemon socket(s) to connect to")
+	flags.VarP(hostOpt, "host", "H", "Daemon socket(s) to bind to")
+	flags.BoolVar(&o.InsecureHostBind, "give-the-internet-root-access", false, "Allow -H to bind to a TCP address without --tlsverify (this could allow anyone on your network to get root access to your *host*)")
```

Comment:
nit: I think we will support rootless mode. Just `--insecure-bind-tcp` seems fine.

Reply:
I don't think `--insecure` is scary enough. Maybe `--allow-internet-facing-remote-code-execution` or something?